composer.json

```diff
@@ -6,7 +6,8 @@
     "require": {
         "php": ">=7.4.0",
         "yiisoft/yii": "^1.1",
-        "sentry/sdk": "^2.1"
+        "sentry/sdk": "3.1",
+        "sentry/sentry": "3.1"
```

Comment:
```suggestion
        "sentry/sdk": "^3.1",
```
By specifying `3.1` directly, it means that if a `3.1.1` bugfix or a `3.2` minor version comes out, we'd have to manually update this in this package, then tag & release it, and _then_ update our usage of it elsewhere.

If we go with the [caret version range](https://getcomposer.org/doc/articles/versions.md#caret-version-range-), it means that we'll be able to get those updates downstream, without having to update this package.

To illustrate, check out the Packagist Semver Checker... I'll use the `2.x` series since there are more options there: [`2.0`](https://semver.mwl.be/#!?package=sentry%2Fsdk&version=2.0&minimum-stability=stable) vs [`^2.0`](https://semver.mwl.be/#!?package=sentry%2Fsdk&version=%5E2.0&minimum-stability=stable) ... by including the caret, we would get any non-breaking updates, so `2.0.0`, `2.0.1`, `2.0.2`, `2.0.3`, `2.0.4`, `2.1.0`, and `2.2.0`... but it would _not_ include the `3.x` line, since it has breaking changes.

It means that further PRs against this repo won't be required until there is significant code changes & testing needed.

---

Also, `sentry/sentry` should be provided by `sentry/sdk`, so we shouldn't have to explicitly require it (removing it also means that we don't have to worry about version conflicts)